apps/playground/src/app/modules/components/select-module/select-module.component.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Angular imports
+ */
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { TagType } from '@betagouv/ngx-dsfr/tag';
+import { SelectOption } from '@betagouv/ngx-dsfr/select';
+
+interface SettingsForm {
+  label: FormControl<string>;
+  description: FormControl<string>;
+  hint: FormControl<string>;
+  disabled: FormControl<boolean>;
+  multiple: FormControl<boolean>;
+  showSelectedOptions: FormControl<boolean>
+  hasFailed: FormControl<boolean>;
+  failureMessage: FormControl<string>;
+  hasSucceeded: FormControl<boolean>;
+  successMessage: FormControl<string>;
+}
+
+interface SimulationForm {
+  select: FormControl<string[]>;
+}
+
+@Component({
+  templateUrl: './select-module.component.html',
+  styleUrls: ['./select-module.component.scss']
+})
+export class SelectModuleComponent implements OnInit {
+
+  settingsForm: FormGroup<SettingsForm> | undefined;
+  simulationForm: FormGroup<SimulationForm> | undefined;
```

Comment:
Instead we define the forms directly here, without typing them explicitly, as Angular and TypeScript will do it for us : 
```typescript
settingsForm = this.formBuilder.group({
    label: ['This is a label', [Validators.required]],
    defaultPlaceholder: ['This is a placeholder', [Validators.required]],
    hint: ['This is a hint', [Validators.required]],
    disabled: [false, [Validators.required]],
    multiple: [false, [Validators.required]],
    showSelectedValues: [false, [Validators.required]],
    hasFailed: [false, [Validators.required]],
    failureMessage: ['This is a failure message', [Validators.required]],
    hasSucceeded: [false, [Validators.required]],
    successMessage: ['This is a success message', [Validators.required]]
  });
  simulationForm = this.formBuilder.group({ select: [['Option 3']] });
```